src/Migration.php

```diff
@@ -1119,6 +1119,26 @@ public function addConfig($values, $context = null)
         return $this;
     }
 
+    /**
+     * Remove configuration value(s) to current context; @see Migration::removeConfig()
+     *
+     * @since 10.1.0
+     *
+     * @param array  $values  Value(s) to remove
+     * @param string $context Context to remove on (optional)
+     *
+     * @return Migration
+     */
+    public function removeConfig($values, $context = null)
+    {
+        $context = $context ?? $this->context;
+        if (!isset($this->configs[$context])) {
+            $this->configs[$context] = [];
+        }
+        $this->configs[$context] += (array)$values;
+        return $this;
+    }
```

Comment:
This is the exact same as `addConfig`. I am not sure it will work as expected.
A test case on this new method is required.